username-password-auth/main.css

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I don't think this file is used. remove it? The one that is being served is the main.css in the `public` directory.